Configure

```diff
@@ -940,12 +940,12 @@ if (defined($config{api}) && !exists $apitable->{$config{api}}) {
         die "***** Unsupported api compatibility level: $config{api}\n",
 }
 
-if (keys %deprecated_options)
+if (sort keys %deprecated_options)
         {
         warn "***** Deprecated options: ",
                 join(", ", keys %deprecated_options), "\n";
         }
```

Comment:
The sort in line 943 does not seem to be necessary. In particular, since the keys are printed unsorted in line 946.